cip-0005.md

```diff
@@ -0,0 +1,81 @@
+<pre>
+  CIP: 5
+  Title: Voting Meta Protocol through Broadcasts
+  Authors: Ruben de Vries
+  Discussions-To: https://counterpartytalk.org/t/cip-proposal-voting-meta-protocol-through-broadcasts/1926
+  Status: Draft
+  Type: Standards
+  Created: 2016-03-21
+</pre>
+
+## Abstract ##
+
+Any user should be able to initiate a vote for all holders of a Counterparty asset.
+Holders of the asset can broadcast their vote and all Counterparty nodes can tally the votes.
+
+## Motivation ##
+
+With the lack of PoW 'voting' by miners there's no good way to reach / poll for consensus on changes to the protocol
+that aren't completely uncontroversial.
+Using this proposed protocol, Counterparty developers can initiate a vote to all XCP holders to learn the will of the community.
+
+## Rationale ##
+
+Being able to let all XCP owners vote on a particular issue can be very useful to poll the consensus / desire of the community.
+
+The votes done in this way is not part of this CIP and the initial intention is not to have any protocol changes activate automatically based on these votes,
+though that could be a future CIP if there's a desire to do so.
```

Comment:
it's markdown, unless I put `  ` at the end of a line it won't show as a line break and it's easier to read/edit the source this way ;)